docs/docs/guides/labs/dg/multiple-projects.md

```diff
@@ -86,8 +86,13 @@ Now we have two projects. We can list them with:
 
 ## Load workspace with `dg`
 
-Finally, let's load up our two projects with `dg dev`. `dg dev` will automatically recognize the projects in your workspace and launch them in their respective environments. Let's run `dg dev` back in the workspace root directory and load up the Dagster UI in a browser:
+Finally, let's load up our two projects with `dg dev`. `dg dev` will automatically recognize the projects in your workspace and launch them in their respective environments. But since the workspace root does not have an associated Python environment, where will the `dg` executable come from? While it is possible to source the `dg` executable from the environment of either project, the simplest solution is to use `uvx`, which will use a separate ephemeral environment containing `dg`:
 
-<CliInvocationExample contents="cd ../.. && dg dev" />
+<CliInvocationExample contents="cd ../.. && uvx dg dev --from=dagster-dg-cli" />
```

Comment:
tangential to the purpose of this PR: i left one more note in my [friction log here](https://www.notion.so/dagster/dg-components-friction-log-20218b92e46280c9a959c902826e4c32?source=copy_link#20218b92e462804ba4bad6b2c590540b) -- when i went through, i didn't need to `cd ../..`. take or leave it -- this can be part of the broader docs pass in case @neverett wants to move more things around.